gitops/argocd/charts/observability/greptimedb/templates/cluster-db.yaml

```
@@ -0,0 +1,68 @@
+# SPDX-FileCopyrightText: Copyright (C) Nicolas Lamirault <nicolas.lamirault@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+---
+# yaml-language-server: $schema=https://kubernetes-schemas.pages.dev/postgresql.cnpg.io/cluster_v1.json
+apiVersion: postgresql.cnpg.io/v1
+kind: Cluster
+metadata:
+  {{- with .Values.additionalAnnotations }}
+  annotations:
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 0 but found 2

(indentation)